Common.xml

```diff
@@ -3519,13 +3519,13 @@ Using values in range 0-127 is recommended to avoid ambiguities with byte order
       </Item>
       
       <Item ID="6049">    
-        <Name>Reserved</Name>     
-        <Operations></Operations>     
-        <Type></Type>
-        <RangeEnumeration></RangeEnumeration>
-        <Units></Units>
-        <Submitter></Submitter>
-        <Description></Description>
+        <Name>Measurement Quality Level</Name>     
+        <Operations>R</Operations>     
+        <Type>Integer</Type>
+        <RangeEnumeration>0..100</RangeEnumeration>
+        <Units>%</Units>
+        <Submitter>Vaisala</Submitter>
+        <Description><![CDATA[Measurement quality level reported by a smart sensor. Value 100 means that the measured value is good, while value 0 means a rejected value that should not be used. Values between 0 and 100 indicate that the measured value may have quality issues. Smaller values indicate bad quality.]]></Description>
```

Comment:
I wonder if we should further qualify “good” and “bad” here. Is there some more formal language used in the measurement community for these things?